Repositories/FeaturesRepository.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AzureStorage;
+using Core.Repositories;
+
+namespace Repositories
+{
+    /// <summary>
+    /// RowKey = featureName
+    /// PartitionKey = clientId || GlobalSettings (const)
+    /// </summary>
+    public class FeaturesRepository : IFeaturesRepository
+    {
+        public const string TableName = "Features";
+
+        public const string GlobalSettingsPartitionKey = "GlobalSettings";
+        
+        private readonly INoSQLTableStorage<FeatureEntry> _tableStorage;
+
+        public FeaturesRepository(INoSQLTableStorage<FeatureEntry> tableStorage)
+        {
+            _tableStorage = tableStorage;
+        }
+
+        public async Task AddOrUpdate(string featureName, bool value, string clientId = null)
```

Comment:
it's not needed in this service. Will be implemented in BO

Reply:
`+`